internal/pkg/provider/data/schema.json

```diff
@@ -10,22 +10,33 @@
       "minimum": 2048
     },
     "architecture": {
-      "enum": ["amd64", "arm64"]
+      "enum": [
+        "amd64",
+        "arm64"
+      ]
     },
     "disk_size": {
       "type": "integer",
-      "minimum": 5,
+      "minimum": 10,
       "description": "In GB"
     },
+    "storage_class_name": {
+      "type": "string",
+      "description": "name of the kubernetes storage class used for volumes"
+    },
     "network_binding": {
-      "enum": ["bridge", "passt"],
+      "enum": [
+        "bridge",
+        "passt"
+      ],
       "default": "bridge"
     }
   },
   "required": [
     "cores",
     "memory",
     "architecture",
-    "disk_size"
+    "disk_size",
+    "storage_class_name"
```

Comment:
I don't think we should make it mandatory. You can have a default storage class and it will work without setting it explicitly